tajo-core/src/main/java/org/apache/tajo/worker/TajoWorker.java

```diff
@@ -242,27 +210,10 @@ private void initWorkerMetrics() {
     workerSystemMetrics = new TajoSystemMetrics(systemConf, Node.class, workerContext.getWorkerName());
     workerSystemMetrics.start();
 
-    workerSystemMetrics.register(Node.QueryMaster.RUNNING_QM, new Gauge<Integer>() {
-      @Override
-      public Integer getValue() {
-        if(queryMasterManagerService != null) {
-          return queryMasterManagerService.getQueryMaster().getQueryMasterTasks().size();
-        } else {
-          return 0;
-        }
-      }
-    });
-
-    workerSystemMetrics.register(Node.Tasks.RUNNING_TASKS, new Gauge<Integer>() {
-      @Override
-      public Integer getValue() {
-        if(taskExecutor != null) {
-          return taskExecutor.getRunningTasks();
-        } else {
-          return 0;
-        }
-      }
-    });
+    workerSystemMetrics.register(Node.QueryMaster.RUNNING_QM, () ->
+        queryMasterManagerService != null ? queryMasterManagerService.getQueryMaster().getQueryMasterTasks().size() : 0);
```

Comment:
This line is over 120 char
https://cwiki.apache.org/confluence/display/TAJO/Code+Style